app/mm-redux/types/module.d.ts

```diff
@@ -4,6 +4,7 @@ declare module 'gfycat-sdk';
 declare module 'remote-redux-devtools';
 declare module 'redux-action-buffer';
 declare module 'react-intl';
+declare module 'redux-mock-store';
```

Comment:
Couldn't install this package because of markdown package issue (Even with `npm i --force` and upgrading to latest node & rpm versions). I had to delete the package-lock.json to make it work.

Reply:
you should try again